content/_index.md

```diff
@@ -25,16 +24,16 @@ carousel:
             Define infrastructure in JavaScript, TypeScript, Python, Go, or
             any .NET language, including C#, F#, and VB.
         - title: Your favorite tools.
-          description: Use familiar IDEs, test frameworks, and tools.
+          description: Use familiar IDEs, test frameworks, and tools. Resource mocking for easy verification.
         - title: Share and reuse.
-          description: Codify best practices and share them with your team or a growing community of practitioners.
+          description: Codify best practices and policies then share them with your team or community as self-service architectures.
```

Comment:
Hm, I don't quite follow the "self-service architectures" point. What are we trying to say?

Reply:
I think we've drifted off of the messaging - can we snap to the descriptions here on the Press Release and let me know if a shorter version is needed to fit the space?

Reply:
Link to PR in slack

Reply:
I'm still having trouble parsing this. What does "sharing ... as self-service architectures" mean?

Reply:
This should have been addressed in the last round of edits. Not sure where this language came from but it's gone in the latest preview...